cypress/integration/tests/rbac/migrator.test.ts

```diff
@@ -30,6 +31,21 @@ describe("Migrator RBAC operations", () => {
         getRandomCredentialsData(CredentialType.sourceControl)
     );
 
+    const rbacRules: RbacValidationRules = {
```

Comment:
Can we create this data as migrator.json under fixtures like analysis.json ?

Reply:
I am going to do this, it will be the next step. PR is in progress yet :-)